golang/cosmos/x/vstorage/types/types.go

```diff
@@ -1,9 +1,68 @@
 package types
 
+import (
+	"encoding/json"
+	"fmt"
+)
+
 func NewData() *Data {
 	return &Data{}
 }
 
 func NewChildren() *Children {
 	return &Children{}
 }
+
+type StorageEntry struct {
+	path  string
+	value *string
+}
+
+func NewStorageEntry(path string, value string) StorageEntry {
+	return StorageEntry{path, &value}
+}
+
+func NewEmptyStorageEntry(path string) StorageEntry {
+	return StorageEntry{path, nil}
+}
```

Comment:
This is going to be alphabetically out of place, which works but is unusual. Did you consider instead changing the signature of `NewStorageEntry` to `(path string, value *string)`?